lms/djangoapps/teams/static/teams/js/views/team_profile_header_actions.js

```diff
@@ -9,7 +9,7 @@
         function (Backbone, _, gettext, TeamUtils, teamProfileHeaderActionsTemplate) {
             return Backbone.View.extend({
 
-                errorMessage: gettext("An error occurred. Try again."),
+                errorMessage: _.escape(gettext("An error occurred. Try again.")),
```

Comment:
`errorMessage` will be placed inside `<div class="copy"></div>` in [teams_tab.underscore](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/teams/static/teams/templates/teams_tab.underscore), so i am not sure what changes i should make in template?